src/ProtectorServiceProvider.php

```diff
@@ -28,7 +28,11 @@ public function boot(): void
         $this->loadMigrationsFrom(__DIR__ . '/../src/Migrations');
```

Comment:
Our intention is, to let the User explicitly publish the migration, since there are also use-cases which do not rely on a user based encryption for example.

Using the `$this->loadMigrationsFrom(__DIR__ . '/../src/Migrations');` should load and run the migrations right from the vendor-directory without the need to publish the migration. See: [Laravel Package Development - Migrations](https://laravel.com/docs/8.x/packages#migrations)

This would lead to cases where a application may crash if the `users`-table does not exist in the applications database or once the migration gets published afterwards and executed (again).